cmd/openshift-install/create.go

```diff
@@ -197,6 +198,10 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		if err != nil {
 			logrus.Fatal(err)
 		}
+		if cmd.Name() != "cluster" {
+			logging.LogCreatedFiles(cmd.Name(), rootOpts.dir, targets)
+		}
```

Comment:
just let each command run it imo. no need to make it this generalized.

Reply:
As the code currently stands, during a run of `create cluster` an INFO message appears that says manifests are created in <dir>. From e2e-aws:

```
time="2020-06-16T12:43:09Z" level=info msg="manifests created in: /tmp/installer/manifests and /tmp/installer/openshift"
```

I do not think we should be displaying this message as the manifests are consumed...